src/pages/editor/EditorInstitution.tsx

```diff
@@ -153,14 +153,16 @@ export const EditorInstitution = () => {
             </Typography>
           </Grid>
 
-          <Grid item xs={12}>
-            <Typography variant="h3" component="h2">
-              {t('editor.institution.institution_support')}
-            </Typography>
-            <Link href={customer?.serviceCenterUri} target="_blank" rel="noopener noreferrer">
-              {customer?.serviceCenterUri}
-            </Link>
-          </Grid>
+          {customer?.serviceCenter?.uri && (
+            <Grid item xs={12}>
+              <Typography variant="h3" component="h2">
+                {t('editor.institution.institution_support')}
+              </Typography>
+              <Link href={customer?.serviceCenter?.uri} target="_blank" rel="noopener noreferrer">
+                {customer?.serviceCenter?.uri}
```

Comment:
sjekkes på linje 156
```suggestion
              <Link href={customer.serviceCenter.uri} target="_blank" rel="noopener noreferrer">
                {customer.serviceCenter.uri}
```